### PR TITLE
feat: Added a pr template specifically for npm/clientside repos #184076

### PR DESCRIPTION
**Pull request by tom-shephard**
# Summary

Added a directory called `.github` which contains a new PR template that has a checklist more tailored for npm / clientside repos

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- ✔ No new security vulnerabilities introduced
+- ❌ New security vulnerabilities found
+- ❎ No new dependencies or changes affecting security
+
+### ESLint rules verified
+- ✔ All new/modified ESLint rules are verified and documented
+- ❌ New/modified ESLint rules are not verified or documented
+- ❎ No changes to ESLint rules
+
+### Playwright configurations verified
```
Comment (richardb355) [inline .github/pull_request_template.md]: This particular repo won't need anything about Playwright.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- ✔ Code is linted and formatted according to the project's guidelines
+- ❌ Code is not linted or formatted correctly
+- ❎ No code changes
+
+### Security vulnerabilities checked
+- ✔ No new security vulnerabilities introduced
+- ❌ New security vulnerabilities found
+- ❎ No new dependencies or changes affecting security
+
+### ESLint rules verified
```
Comment (richardb355) [inline .github/pull_request_template.md]: What does this one mean? Is it not covered by the section on linting?

**Review by richardb355 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (tom-shephard) [reply .github/pull_request_template.md]: I don't understand sorry. What other section on linting?

File: .github/pull_request_template.md (hunk 1 above)
Comment (tom-shephard) [reply .github/pull_request_template.md]: Good point. I have removed this from this one and added a separate Playwright template to that repo here:
https://github.com/audaciaconsulting/Audacia.Testing.Playwright.Helper/pull/2

File: .github/pull_request_template.md (hunk 2 above)
Comment (tom-shephard) [reply .github/pull_request_template.md]: Apologies. I don't know why I didn't see that before. I have removed as I agree it's not necessary to have a second section on ES Linting alone

**Review by richardb355 — APPROVED** [review]

**Pull request merged**